pom.xml

```diff
@@ -46,31 +46,42 @@
   <properties>
     <!-- Defaut compiler settings -->
     <maven.compiler.optimize>true</maven.compiler.optimize>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
     <!-- eXo Modules -->
-    <org.exoplatform.depmgt.version>15</org.exoplatform.depmgt.version>
-    <org.exoplatform.platform.version>5.2.0</org.exoplatform.platform.version>
-    <!-- for tests -->
-    <junit.version>4.12</junit.version>
-    <org.mockito.version>2.2.11</org.mockito.version>
+    <org.exoplatform.social.version>6.0.0-M18</org.exoplatform.social.version>
```

Comment:
This stuff not for master yet. It's for develop branch where we already upgraded to 6.0.0-M016. Master is for 5.2.x line and will be rolled soon to the stable branch.

Reply:
ok, thanks Peter. The base was changed to develop of the PR.
FYI, as mentioned in description, it's a prepared PR to be applied once 6.0.0-M18 gets released:
> To apply after releasing 6.0.0-M18

I think we should rename the `master` branch to `5.2.x`, and make `develop` as default branch (same strategy that we used for all other addons). To do so, I suggest to open an ITOP issue and notify @vsellier to make the modification.